src/index.ts

```diff
@@ -147,7 +152,8 @@ export default class MachinomyServerPlugin extends MiniAccountsPlugin {
         accountName,
         balance: new BigNumber(0)
       }, {
-        set (obj, prop, val) {
+        set: (obj, prop, val) => {
+          obj[prop] = val
```

Comment:
This is redundant with the `Reflect.set` call. I'm also not familiar enough with Proxy/Reflect to know why this doesn't cause an infinite loop when we set inside of the set handler, but if the tests work then it must not be an issue

Reply:
Will change to return true instead of the Reflect.set call -- forgot to update that. Previously, the object property was not set prior to putting the object in the store (thus storing an old object).

I'm not too familiar with proxies either, so I'm not sure why it doesn't cause an infinite loop. In the MDN example linked below, Reflect.set is used within the set handler of the proxy. Yet the interceptions section beneath the example states that Reflect.set triggers the trap as well.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/handler/set